### PR TITLE
131 accout type in signup page

### PR DESCRIPTION
**Pull request by tanmaythole**

### CODE REVIEW

File: src/Components/Auth/Signup/AccountTypeCard.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Card } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+import "./style.css";
+
+const AccountTypeCard = ({ data, accountType, setAccountType }) => {
+  const { Icon, title, subTitle, value } = data;
+
+  const isActive =
```
Comment (tanmaythole) [inline src/Components/Auth/Signup/AccountTypeCard.jsx:12]: add comment

File: src/Layout/AuthLayout/index.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import PropTypes from "prop-types";
 const AuthLayout = ({ children }) => {
   return (
```
Comment (tanmaythole) [inline src/Layout/AuthLayout/index.jsx]: return not required

File: src/Components/Auth/Signup/AccountTypeCard.jsx (hunk 1 above)
Comment (tanmaythole) [reply src/Components/Auth/Signup/AccountTypeCard.jsx:12]: 👍🏻

File: src/Layout/AuthLayout/index.jsx (hunk 2 above)
Comment (tanmaythole) [reply src/Layout/AuthLayout/index.jsx]: 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**